### ISSUE TITLE
Re-Word "Supress Referrers" Warning

### ISSUE THREAD

**Issue by ninavizz** [freedomofpress/securedrop.org#637]
Gang... this warning is alarmist and means nothing to me. The link just takes me to a technical page that means less to me. I still don't know what it means for a page to _suppress a referrer_, nor why I should care. On an aside, the sentence structuring is also stumbly and far too many words seem to be getting used to communicate what feels like a simple concept. I think I might grok what's being gotten at after reading it a few times, but it badly needs improvement.

![image](https://user-images.githubusercontent.com/8262612/61818166-1e6c6300-ae05-11e9-91f8-26dfc355f02b.png)

1. Warnings should first and foremost tell users what they should do. The Tor Browser notation is a solution, not what the person should do. The word "Caution" preceding all other text, is the first thing to come to mind... as the behavior you're seeking of users, is to proceed with caution.

2. What is a plain-language TL;DR... my anonymity may be compromised, my computer might get hacked, what? What is the threat to me in following the below link. The language "suppress a referrer" sounds very tertiary to what needs to be communicated to a user at the level of the alert itself.

3. Developer documentation should never, ever be the destination for "Learn More" type links, from error messages. Unless the message itself is on a page that a developer or IT person would be consuming content on.

Let's work on something clearer?

**Comment by eloquence** [comment]
Thanks for flagging, but I would ask to defer an work on this for now. This only impacts a small number of currently listed entries, and we're generally not permitting new entries to be listed if they have this problem.

**Comment by ninavizz** [comment]
Agreed wrt priorities. Only wanted to create the Issue to get the dealie-o flagged and on the record as a Nina Finger-Wag™
